sdk/include/opentelemetry/sdk/common/attribute_utils.h

```diff
@@ -114,11 +114,21 @@ class AttributeMap
         });
   }
 
+  const std::size_t GetAttributeMapSize() const noexcept
```

Comment:
I don't think we need this. Please check my PR here, as it'd be nicer if we simply expose all properties of real `std::map`-alike container here:

https://github.com/open-telemetry/opentelemetry-cpp/blob/752c5251734a90386d857928eeb73bea53926ff1/sdk/include/opentelemetry/sdk/common/attribute_utils.h#L102

Reply:
I will pick your changes from attribute_utils.h and modify accordingly.

Reply:
Sounds good. That works. I'll reset my status to unblock your review.